src/modules/file/handlers/storage/storage.file.handler.ts

```diff
@@ -131,7 +153,7 @@ export class StorageFileHandler implements IStorageFileHandler {
     return FileType.OTHER;
   }
 
-  private getPath(input: ReadFileDto, isPartial: boolean) {
+  public getPath(input: ReadFileDto, isPartial: boolean) {
```

Comment:
This is private because where the file is stored is a detail that should be known by the handler but should not be required externally. It could be a url, a path, a hash, etc.